src/storage/copy_arrow/copy_structures_arrow.cpp

```diff
@@ -320,6 +320,62 @@ std::unique_ptr<Value> CopyStructuresArrow::getArrowList(std::string& l, int64_t
         DataType(LIST, std::make_unique<DataType>(childDataType)), std::move(values));
 }
 
+std::unique_ptr<uint8_t[]> CopyStructuresArrow::getArrowTensor(std::string& l, int64_t from,
```

Comment:
Can we not reuse getArrowList?

Reply:
There are some differences between fixed/var list for now:
1. getArrowVarList returns a vector of values, and then convert that vector to ku_list_t, and store the elements in the overflow.
3. getArrowFixedList should return a pointer to a block of memory which stores the elements of the list. We should copy that block of memory to inMemColumn directly.

Reply:
Got it. Let's try to reuse the part of parsing lists.